src/create.rs

```diff
@@ -50,27 +60,27 @@ pub fn create(args: InputArgs) {
         process::exit(1);
     });
 
-    let mut clone = GitClone {
-        remote_url: remote_url.as_str(),
-        local_path: root_path.as_path(),
+    let clone = GitClone {
+        remote_url,
+        local_path: root_path,
         use_ssh: false,
     };
 
-    clone.git_action().expect(format!("Failed to clone repo {}, ", remote_url).as_str());
+    let multi_bars = ProgressTracker::new();
```

Comment:
I am not able to see a progress bar for create. In case it takes time there is no output shown. Only the final result after clone is printed.

Reply:
It might need a bit more refactor : the progress bar is made for tracking things that happen in a `GitAction`, and most of the creation step happen from outside. Either my progress system must be more flexible, either we need to put everything in a `GitAction`

Reply:
The wait is actually when cloning after remote creation is done. If we add a line for clone before download begins, it should be fine.